templates/typescript/samples/sample-assistant/src/index.ts

```diff
@@ -136,9 +132,23 @@ supportedLocales.forEach((locale: string) => {
 
     localizedTemplates.set(locale, localeTemplateFiles);
 });
-    
+
 const localeTemplateEngine: LocaleTemplateEngineManager = new LocaleTemplateEngineManager(localizedTemplates, botSettings.defaultLocale || 'en-us')
 
+// Create the skills configuration class
+let authConfig: AuthenticationConfiguration;
+let skillConfiguration: SkillsConfiguration;
+if ((skills !== undefined && skills.length > 0) && (skillHostEndpoint !== undefined && skillHostEndpoint !== "")) {
```

Comment:
```suggestion
if (skills !== undefined && skills.length > 0 && skillHostEndpoint.trim().length !== "") {
```

As all the validations are `&&`